src/main/java/com/groupthree/culinarycompanion/dto/RecipeCollectionDTO.java

```diff
@@ -6,7 +6,6 @@ public class RecipeCollectionDTO {
     private int collectionId;
     private int userId;
     private List<Integer> recipeIds;
-    // Getters and setters
```

Comment:
Removing this as it is unnecessary
Reference: https://www.adservio.fr/post/java-code-review-checklist